packages/service-library/src/servicelib/archiving_utils.py

```diff
@@ -347,6 +391,8 @@ def prune(self, exclude: set[Path]) -> None:
 
 __all__ = (
     "archive_dir",
+    "ArchiveError",
+    "is_leaf_path",
```

Comment:
there I do have a question: if we use the pattern with private functions starting with ```_```, does the __all__ tuple still make any sense?

Reply:
``__all__`` is actually necessary when you do ``from module import *``. 
We do not use that import method but I find that ``__all__`` is a nice way to highlight/document what is the functions that the developer expects to be "importable", or in other words, the actual API of the module